pkg/controllers/provisioning/v1alpha1/reallocation/annotate.go

```diff
@@ -52,7 +52,8 @@ func (a *Annotator) ClearUnderutilized(ctx context.Context, nodes []*v1.Node) er
 			return fmt.Errorf("listing pods on node %s, %w", node.Name, err)
 		}
 
-		if !utilsnode.IsUnderutilized(ptr.PodListToSlice(pods)) {
+		if !utilsnode.IsUnderutilized(ptr.PodListToSlice(pods),
```

Comment:
Would it be cleaner to these details from this scope? 
```utilsnode.IsUndertuilized(pods,nodes)```

Reply:
I think it would ideally be cleanest to have a form like that. I had it separated like this because I hadn't decided if I wanted to make it possible to specify both a percentage and a raw number. After sleeping on it, I think it makes more sense just to use percentage! Thanks.